node/src/index.ts

```diff
@@ -426,6 +426,16 @@ export class LocalTable<T = number[]> implements Table<T> {
     return new Query(query, this._tbl, this._embeddings)
   }
 
+  /**
+   * Creates a filter query to find all rows matching the specified criteria
+   * @param value The filter criteria (like SQL where clause syntax)
+   */
+  filter (value: string): Query<T> {
+    return new Query(undefined, this._tbl, this._embeddings).filter(value)
```

Comment:
Does `table.filter("id != 2")` only return 10 results?  This is creating a brand new `Query` and I think in the constructor  `_limit` defaults to 10.  As a user, this wouldn't be obvious from reading the description of this method.

Reply:
nice catch. I removed the default limit if the vector query is not specified. This way users aren't surprised later on. Would you mind taking another look?